app/component/EndLeg.js

```diff
@@ -35,7 +35,7 @@ function EndLeg(props, context) {
         </div>
       </div>
       <div className={`leg-before ${modeClassName}`} aria-hidden="true">
-        <div className={`leg-before-circle circle ${modeClassName}`} />
+        <div className={`leg-before-circle el circle ${modeClassName}`} />
```

Comment:
Is this on purpose? If "el" is supposed to signify "end leg", let's use `end-leg`.

Reply:
No it is not, wil remove it again